core/icon.js

```diff
@@ -47,31 +48,31 @@ Blockly.Icon = function(block) {
 /**
  * Does this icon get hidden when the block is collapsed.
  */
-Blockly.Icon.prototype.collapseHidden = true;
+Icon.prototype.collapseHidden = true;
 
 /**
  * Height and width of icons.
  */
-Blockly.Icon.prototype.SIZE = 17;
+Icon.prototype.SIZE = 17;
```

Comment:
Does this need an `@const` annotation? It's weird b/c it's defined on the prototype, so idk.

Reply:
It doesn't appear to be mutated anywhere and the compiler/tests were happy, so I added it.